bower.json

```diff
@@ -33,25 +33,24 @@
     "backbone": "1.3.3",
     "underscore": "1.8.3",
     "bootstrap": "3.3.7",
-    "chance": "1.0.3",
+    "chance": "1.0.8",
     "d3": "3.5.16",
-    "visionmedia-debug": "2.2.0",
```

Comment:
debug no longer comes with a distribution (instead see the UPDATE at `/src/client/lib/debug` same as superagent)